torchaudio/backend/common.py

```diff
@@ -124,6 +144,9 @@ def __init__(self,
 It assumes that the wav file uses 16 bit per sample that needs normalization by
 shifting the input right by 16 bits.
 
+Warnings:
+    This function is deprecated. Please migrate to the new backend and use ``load(.., normalize=False)``
```

Comment:
"... is deprecated and will be removed in a future release."

"New backend" should also be more specific here.